neqo-transport/src/connection.rs

```diff
@@ -1099,8 +1099,16 @@ impl Connection {
             }
         };
         let mut builder = if pt == PacketType::Short {
+            qdebug!("Building Short dcid {}", &path.remote_cid,);
```

Comment:
Want to add `[self]` ?

Reply:
build_packet_header doesn't take a `self` argument, and adding one causes ownership issues. :frowning_face:

Reply:
I opened #439 as a follow-on for this comment.